scripts/software/apps.sh

```diff
@@ -1,40 +1,243 @@
 #!/bin/bash
 
+# Check for an app and if it exists provide the version.
+function check() {
+    name=$1
+    version=$2
 
-# check for a app and if exists provide the version
-function check {
-	name=$1
-	version=$2
+    key=$name
 
-	# add the prefix app_ to every entry
-	key="app_$name"
-
-	if command -v "$name" > /dev/null; then
-		echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
-	else
-		echo VALUE BAS "$key" \""<not found>"\"	GROUP "missing_apps"
-	fi
+    if command -v "$name" >/dev/null; then
+        echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
+    fi
 }
 
+'''
+Shells.
+'''
 
 check "bash" "-version"
+check "zsh" "--version"
+check "csh" "--version"
+check "ksh" "--version"
+check "fish" "--version"
+
+'''
+Code Helpers.
+'''
+
+# Shell Parser, Formatter and Interpreter.
+check "shfmt" "--version"
+# Regex Evaluator.
+check "kiki" "--version"
+# Flash Card Tool to Help Memorize Shortcuts.
+check "mnemosyne" "--version"
+# Fix Previous Command.
+check "thefuck" "--version"
+# Hex Editor.
+check "ghex" "--version"
+# GUI Diff.
+check "meld" "--version"
+# Code Indexing.
+check "ctags" "--version"
+# Code Search (a specialized grep).
+check "ack" "--version"
+
+'''
+Terminal Management.
+'''
+
+# Terminal Multiplexer.
+check "byobu" "--version"
+# Another Terminal Multiplexer.
+check "tmux" "--version"
```

Comment:
[man tmux](http://man7.org/linux/man-pages/man1/tmux.1.html)

> -V            Report the tmux version.

Reply:
What I get for being lazy and not checking them up :)
Thanks for pointing these out.

Reply:
:heavy_check_mark: